src/middlewares/check_auth.rs

```diff
@@ -54,7 +54,7 @@ fn is_auth_route(routes: &[String], path: &str) -> bool {
 }
 
 #[derive(Debug)]
-pub struct UserId(String);
+pub struct UserId(pub String);
```

Comment:
I think that this `pub` is not needed. You can access `UserId.0`

Reply:
Without the `pub` you can't access the `.0`